data/model/organization_skus.py

```diff
@@ -0,0 +1,73 @@
+import logging
+
+import peewee
+
+from data import model
+from data.database import OrganizationRhSkus
+
+logger = logging.getLogger(__name__)
+
+
+def get_org_subscriptions(org_id):
+    subscriptions = []
+    try:
+        query = OrganizationRhSkus.select().where(OrganizationRhSkus.org_id == org_id).dicts()
```

Comment:
Could you return the query object itself here, and let the caller deal with that?
That way the results get lazily loaded only when necessary.